src/OrchardCore.Themes/TheAdmin/Views/UserMenu.cshtml

```diff
@@ -0,0 +1,8 @@
+<ul class="nav navbar-nav">
+    <li class="nav-item">
+        <a class="nav-link"><span><i class="fa fa-user" aria-hidden="true"></i></span><span> @User.Identity.Name</span></a>
+    </li>
+</ul>
+<form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post" class="form-inline">
+    <button type="submit" class="btn btn-link nav-link" role="button"><i class="fa fa-sign-out" aria-hidden="true"></i> @T["Log off"]</button>
+</form>
```

Comment:
This is exactly the same as the original code, right?

Reply:
Ya, I just move it into another file, so it can be customizable on any theme